examples/mobile/UIComponents/components/controllers/add-delete-button.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta content="width=device-width, user-scalable=no" name="viewport" />
+	<link href="../../lib/tau/mobile/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../css/style.css" rel="stylesheet" />
+	<script data-build-remove="false" src="../../lib/tau/mobile/js/tau.js">
+	</script>
+</head>
+
+<body>
+	<div class="ui-page" id="switch-demo">
+		<header>
+			<div class="ui-appbar-left-icons-container">
+				<a href="#" class="ui-btn ui-btn-icon ui-btn-icon-back" data-style="flat" data-rel="back"></a>
+			</div>
+			<h1>
+				Add/Delete Button
+			</h1>
+		</header>
+		<div class="ui-content">
+			<ul class="ui-listview ui-content-area">
+				<li class="ui-li-divider">
+					<div class="ui-li-text">
+						Add
+					</div>
+					<div class="ui-li-action">
+						<a href="#" class="ui-btn ui-btn-icon ui-icon-add" data-style="flat" ></a>
```

Comment:
instead of `ui-icon-add` use `data-icon="add"`

Reply:
Please check `data-icon="plus"` `data-icon="minus"`

Reply:
Ok, done